spec/models/step_by_step_page_spec.rb

```diff
@@ -216,6 +216,27 @@
     end
   end
 
+  describe 'should_check_links?' do
+    it 'should return false if step by step is scheduled' do
+      step_by_step_with_step = create(:step_by_step_page, status: 'scheduled', scheduled_at: 1.day.from_now)
+
+      expect(step_by_step_with_step.should_check_links?).to be false
+    end
+
+    it 'should return false if step by step has not been edited since publish' do
```

Comment:
I think this should be renamed to `should return false if the status of the step by step is published`

Reply:
I agree with Leena's comment here, since the test is relying on a published status doing what it should do.